src/CMakeLists.txt

```diff
@@ -133,6 +133,7 @@ ELSE ()
 ENDIF()
 
 IF (USE_HTTPS)
```

Comment:
The comment for `USE_HTTPS` says that you can set it to the name of the backend that you want to use.  I don't think that's true, but it would be nice if it _was_.

Reply:
It does though. If you pass `ON`, this will use "autodetection" (a.k.a. use whatever backend we've found and is first in the following tests), Otherwise, this will be passed verbatim as `HTTPS_BACKEND`, which is handled just below (and will error if you passed nonsense).

Reply:
Yep, rereading, I see it now.  Thanks!